src/components/InputAddress.js

```diff
@@ -0,0 +1,187 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import styled, { keyframes } from "styled-components";
+import Input from "./Input";
+
+const fadeIn = keyframes`
+  0% { transform: translateY(-10px); opacity: 0;}
+  100% { transform: translateY(0); opacity: 1; visibility: visible; }
+`;
+
+const fadeOut = keyframes`
+  0% { transform: translateY(0); opacity: 1; visibility: visible; }
+  100% { transform: translateY(-10px); opacity: 0; visibility: hidden; }
+`;
+
+const SInput = styled(Input)`
```

Comment:
How come you always need to expand input, it should be set to width of 100%, this is inside row directioned container right?